src/controllers/test-suite/test-item-controller.ts

```diff
@@ -110,7 +110,7 @@ export const update = async (req: Request, res: Response) => {
     const user = req.user
 
     const { id } = req.params
-    const { apiId, http_method, payload, response, description } = req.body
+    const { apiId, status, http_method, payload, response, description } = req.body
```

Comment:
move **status** after the **description**

Reply:
done :3